docs/src/pages/guides/styling.md

```diff
@@ -77,7 +77,7 @@ Styling in Astro is meant to be as flexible as you’d like it to be! The follow
 
 ¹ _`.astro` files have no runtime, therefore Scoped CSS takes the place of CSS Modules (styles are still scoped to components, but don’t need dynamic values)_
 
-All styles in Astro are automatically [**autoprefixed**](#cross-browser-compatibility) and optimized, so you can just write CSS and we’ll handle the rest ✨.
+All styles in Astro are automatically [**autoprefixed**](#cross-browser-compatibility), minified, and bundled, so you can just write CSS and we’ll handle the rest ✨.
```

Comment:
Hey Marcus, this should read as:
All styles in Astro are automatically [**autoprefixed**](#cross-browser-compatibility), minified and bundled, so you can just write CSS and we’ll handle the rest 
To be honest I was coming to this file soon for editorial review, I'll make sure your change is reflected in the review.

Reply:
@aFuzzyBear A classic debate, to Oxford comma or not 😄 I strongly prefer with oxford comma as it is less ambiguous (like it is autoprefixed as one thing and then "minified and bundled" as another thing?) but if you don't approve I can remove it
https://en.wikipedia.org/wiki/Serial_comma

Reply:
😂 en gaurde, 

Like with everything quintessentially english there are rules with the prevailing oxford comma. If and is used to juxtapose two prepositions together then the comma comes before and after not in-between....tbh dude you had to of had the english teacher that I had. Dyslexia wasnt allowed in the classroom, else you and it would be spending detention. Slowly correcting whatever inconspicuous grammar correction that it was that sent you in there... 🤦‍♂️

Reply:
Touché, I'll take your word for it 😆. I'm not even a native english speaker so I wouldn't know, I just know that for me personally it makes the intention less ambiguous. Not that it matters in this case and it should be clear for anyone what we mean regardless 😄 I'll remove it.